README.md

```diff
@@ -47,6 +47,11 @@ github-contrib -token=<your-token> <org> <github-handle>
     	print version and exit
 ```
 
+Example: get contribution list for `nikhita` against `kubernetes` community.
+```
+# github-contrib -token=5af802c9dc538c87c593e6fd173fa251a92cbcci kubernetes nikhita
```

Comment:
What token value is this? Hope it's not an active token :grimacing: 

Would prefer to leave the token value as just `<token>` or something.

Reply:
Don't be panic, not an active token. :)

Reply:
Phew :)

Can we still replace this with just `<token>`? Even though it's not an active token, I'm concerned folks could try using it. :/

Reply:
Sure. :)